### PR TITLE
yabai: 5.0.4 -> 5.0.6

### PR DESCRIPTION
**Pull request by gshpychka**
###### Description of changes

Changelog: https://github.com/koekeishiya/yabai/blob/master/CHANGELOG.md#506---2023-05-27

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/darwin/yabai/default.nix [hunk 1]
```diff
@@ -52,7 +52,7 @@ in
 
     src = fetchzip {
       url = "https://github.com/koekeishiya/yabai/releases/download/v${version}/yabai-v${version}.tar.gz";
-      sha256 = "sha256-2PH3Hi9x0323MjKHPybNmFddvNNlsaDb1LdiVcZTNJc=";
+      sha256 = "sha256-1szyjcwkhn2wbrcfhh9lh5bnfm1cavxrx6xj4q7521z3zj29a9kf";
```
Comment (Artturin) [inline pkgs/os-specific/darwin/yabai/default.nix:55]: this the stuff behind sha256- isn't a SRI hash

```
❯ nix build ".#yabai"
error: invalid SRI hash '1szyjcwkhn2wbrcfhh9lh5bnfm1cavxrx6xj4q7521z3zj29a9kf'
(use '--show-trace' to show detailed location information)
```

Comment (Artturin) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: https://github.com/NixOS/nixpkgs/pull/242680

Comment (toonn) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: How did this pass ofborg?

Comment (Kranzes) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: ofborg doesn't test anything for yabai it seems?

Comment (toonn) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: Doesn't ofborg evaluate all changed packages from a PR?

Comment (Kranzes) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: Usually, yes, but it seems the way yabai is exposed is a bit different.

Comment (gshpychka) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: My bad, apologies for the bad PR. I trusted `nix-prefetch-url`.

Comment (Kranzes) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: try `nurl` next time, or `nix-update` directly.

Comment (toonn) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: Would be nice if something could be done about the blindspot for ofborg.

Comment (gshpychka) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: Could this be why `yabai` doesn't show up in search over at [search.nixos.org ](https://search.nixos.org/packages?channel=unstable&type=packages&query=yabai)?

Comment (Kranzes) [reply pkgs/os-specific/darwin/yabai/default.nix:55]: Yes

**Pull request merged**